### DISCUSSION TITLE
How to pass date filter value to query?

### DISCUSSION THREAD

**Discussion by ashwin-nath-m** [pxlrbt/filament-excel#142] [Q&A]
I am using datefilter specifically malzariey/filament-daterangepicker-filter . How can I pass the selected filter value to query?

**Comment by Umar-Farooq-Shafi** [comment]
@ashwin-nath-m I was facing the same issue. The only solution I found was to cache the query.

**Comment by Rahul-codoffer** [comment]
@pxlrbt Plz can any one suggest how to pass data in modifyQueryUsing function
